java_tools/configuration_definition/src/main/java/com/rusefi/ConfigField.java

```diff
@@ -261,7 +259,39 @@ public boolean isFsioVisible() {
         return fsioVisible;
     }
 
-    public String autoscaleSpec() { return this.autoscaleSpec; }
+    public String autoscaleSpec() {
```

Comment:
looks like indentation is wonky here

Reply:
Yeah I just can't win.  The C++ is consistently indented while the Java code is a mixture of tabs (8 space?) and 4 spaces (no tabs?).  Not sure which way to go with Java...

Reply:
our java should be all four-space

Reply:
.....because that's the intellij default

Reply:
Ok updated my emacs rule for java-mode, fixed this function.